bindings/rust/wasmedge-sys/examples/async_run_func.rs

```diff
@@ -0,0 +1,62 @@
+use wasmedge_sys::{Config, Loader, Store, Vm, WasmValue};
```

Comment:
Please add the necessary description of how to build/run this example. If there was any assumption, please also make it clear to users. Thanks!